meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/apalis-imx8/bootloader.uuu.in

```diff
@@ -7,9 +7,22 @@ CFG: FB: -vid 0x0525 -pid 0x402F
 CFG: FB: -vid 0x0525 -pid 0x4030
 CFG: FB: -vid 0x0525 -pid 0x4031
 
+SDPU: delay 1000
+SDPU: write -f u-boot-mfgtool.itb
+SDPU: jump
```

Comment:
Do we need both SDPU and SDPV parts? I might be wrong, but it seems all imx8 variants support SDPV.

Reply:
it's more universal, as we can use the same script for SPL and non-SPL builds